src/opaque.rs

```diff
@@ -224,6 +224,10 @@ where
     Le<<<OprfHash<CS> as CoreProxy>::Core as BlockSizeUser>::BlockSize, U256>: NonZero,
 {
     /// Create [`ServerSetup`] with the given keypair
+    ///
+    /// This function should not be used to restore a previously-existing
+    /// instance of [ServerSetup]. Instead, use [ServerSetup::serialize] and
+    /// [ServerSetup::deserialize] for this purpose.
```

Comment:
```suggestion
    /// instance of [`ServerSetup`]. Instead, use [`ServerSetup::serialize`] and
    /// [`ServerSetup::deserialize`] for this purpose.
```

Reply:
The styling for the rest of the documentation is not using backticks for the structs / functions at the moment, but I'll try changing this style for the entire file to see how it looks and will address these changes in its own PR.